docs/USER_PROFILE_IMAGE_UPLOAD.md

```
@@ -0,0 +1,404 @@
+ON storage.objects FOR UPDATE
+TO authenticated
+USING (
+  bucket_id = 'user-profiles'
+  AND (storage.foldername(name))[1] = 'avatars'
+  AND auth.uid()::text = split_part((storage.filename(name)), '-', 1)
+);
+```
+
+#### 3. Bucket Structure
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)